js_modules/dagit/packages/core/src/assets/AssetEventMetadataEntriesTable.tsx

```diff
@@ -0,0 +1,92 @@
+import {Box, Caption, Colors, Icon, Mono} from '@dagster-io/ui';
+import moment from 'moment';
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+import {MetadataEntry} from '../metadata/MetadataEntry';
+import {titleForRun} from '../runs/RunUtils';
+
+import {AssetMaterializationFragment} from './types/AssetMaterializationFragment';
+import {AssetObservationFragment} from './types/AssetObservationFragment';
+
+/**
+ * This component shows the metadata entries attached to an Asset Materialization or Observation event.
+ * AssetNodes also have definition-time metadata, which is unrelated to this event metadata.
+ */
+export const AssetEventMetadataEntriesTable: React.FC<{
+  event: AssetObservationFragment | AssetMaterializationFragment | null;
+  observations?: (AssetObservationFragment | AssetMaterializationFragment)[];
+}> = ({event, observations}) => {
+  if (!event || (!event.metadataEntries.length && !observations?.length)) {
+    return <Caption color={Colors.Gray500}>No materializations</Caption>;
+  }
+
+  const {metadataEntries, timestamp} = event;
+
+  return (
+    <DetailsTable>
+      <tbody>
+        {metadataEntries.map((entry) => (
+          <tr key={`metadata-${entry.label}`}>
+            <td>
+              <Mono>{entry.label}</Mono>
```

Comment:
This row is mono but the others aren't. I think maybe let's just skip the mono font everywhere in the table?

Reply:
Ahh good catch! I think for now I'll monospace the rest of these actually and ask Josh about it when he gets back. We also show these metadata tables inline in the logs and I think it's nice to have consistency between those two. It's arguable whether the log `info` column should be monospace though... maybe we could undo it in both places :-/